optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -501,7 +501,7 @@ es:
     index:
       # the prefix prepended to all Optimize index and alias names
       # NOTE: Changing this after Optimize was already run before, will create new empty indexes
-      prefix: ${CAMUNDA_OPTIMIZE_ELASTICSEARCH_SETTINGS_INDEX_PREFIX:optimize}
+      prefix: ${CAMUNDA_OPTIMIZE_ELASTICSEARCH_SETTINGS_INDEX_PREFIX:}
```

Comment:
as above, my understanding was that this setting would go away and be replaced by the global prefix? Or instead is the intention just to reuse this config option for the global prefix value?

Reply:
Focus of this PR is just about changing the default value, we will remove this setting in a follow up PR